### PR TITLE
Add links to Ethical Web Principles

### PR DESCRIPTION
**Pull request by rhiaro**
Links to Ethical Web Principles from the abstract and section 1 intro, for #282

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/pull/295.html" title="Last updated on Apr 14, 2021, 3:00 PM UTC (71615e1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/295/5c87ce8...71615e1.html" title="Last updated on Apr 14, 2021, 3:00 PM UTC (71615e1)">Diff</a>

### CODE REVIEW

**Review by torgo — CHANGES_REQUESTED** [review]
consistency in use of [[reference]] - otherwise gtg

**Review by hadleybeeman — APPROVED** [review]
Looks good to me!

File: index.bs [hunk 1]
```diff
@@ -70,6 +70,11 @@ urlPrefix: https://www.w3.org/TR/payment-request/; spec: payment-request
 
 <h2 id="basic-principles">Principles behind design of Web APIs</h2>
 
+The Design Principles are directly informed by the ethical framework set out in the
+Ethical Web Principles [[ETHICAL-WEB]]. These principles provide concrete practical
+advice in response to the higher level ethical responsibilities that come with
+developing the Web platform.
+
```
Comment (cynthia) [inline index.bs:77]: We introduced more aggressive line breaking during a rewrite of this document, would it be possible to break the lines following that convention?

(The rationale was to make diffs easier to follow, but also to make it easier to read)

**Review by cynthia — APPROVED** [review]
LGTM!

**Comment by cynthia** [comment]
Awating on @torgo 's change request. (Has that been done?)

**Review by torgo — APPROVED** [review]

**Pull request merged**